vpr/src/route/rr_graph.cpp

```diff
@@ -785,7 +787,9 @@ void create_rr_graph(const t_graph_type graph_type,
                            directs, num_directs,
                            &det_routing_arch->wire_to_rr_ipin_switch,
                            is_flat,
-                           Warnings);
+                           Warnings,
+                           router_opts
+                           );
```

Comment:
Smill Nitpick, this `);` should not be on a new line. Notice what it looked like before.

Reply:
One of the key things about working on large code-bases is being consistent. Although there are tens of people working on code that has been around for decades, it is important that the code feels like it was coded by one programmer.